### ISSUE TITLE
Dont understand migratedTo

### ISSUE THREAD

**Issue by CapablancaYEHA** [gram-js/gramjs#408]
Hello everyone! I use gramjs with TS

I call messages.MigrateChat and response is Update object, which includes 
`chats: {
    migratedTo: {
        channelId: BigInt ?
        accessHash: BigInt ?
    } 
...
}[ ]
`
if console.logged is shows something like
`
migratedTo: Integer { value: 12123422n }
`

So, lets say i check my  Basic group via bot get id 1507129XXX
Then i migrate it to SuperGroupand in repsonse new id is 1662016XXXn as BigInt
But again if i check this SuperGroup via bot, ID is -1001662016XXX

Methods like messages.GetFullChat and channels.GetParticipants is said to accept channelId as bigInt, but actually only -1001662016XXX works.

If i try to convert with 1662016XXXn i never get 1001662016XXX and vise versa (i mean BigInt and Number constructors)
So whats the point to give me a response with bigInt if in reality i should just call toString on it, remove n and add '100' in the beginning ?

Am i missing something? Like in some point at time those BigInts in TG will really be "true" ? So i should prepare my code for that or what?

TL:DR
migratedTo returns BigInt which Does Not represent real ID, you can search group in TG with

**Comment by painor** [comment]
Bot API adds -100 in front of supergroup IDs to represent that they are group ids not user ids.

The real ID is the part after -100. 
All new IDs are BigInts and will not fit a normal number

**Comment by CapablancaYEHA** [comment]
@painor thank you for commitment, but i dont quite understand still.

1) channels.GetFullChannel accepts id as  -100X number, not XXXn BigInt notation. As with other several methods.
2) Do you have any best practises in your expirience? Should i really store chat IDs as BigInt (with Xn notation) or is it okay that i store them as simple Number?

**Comment by CapablancaYEHA** [comment]
I thought it would be better to continue discussion in gramjschat in TG
still didnt quite understand your recommendations
